glue-codes/openfast-cpp/src/OpenFAST.cpp

```diff
@@ -822,8 +832,10 @@ void fast::OpenFAST::allocateMemory() {
     cDriver_Input_from_FAST.resize(nTurbinesProc) ;
     cDriver_Output_to_FAST.resize(nTurbinesProc) ;
     
-  if(!scStatus)
-      scio.from_SC.resize(nTurbinesProc);
+    if(scStatus) {
+        std::cout << "Use of Supercontroller is not supported through the C++ API right now" << std::endl;
```

Comment:
Couldn't this whole if statement be removed anyway? This gets resized in the supercontroller::init, too.